http-webhook.md

```diff
@@ -0,0 +1,339 @@
+# HTTP 1.1 Web Hooks for Event Delivery
+
+## Abstract
+
+"Web hooks" are a popular pattern to deliver notifications between applications
+and via HTTP endpoints. In spite of pattern usage being widespread, there is no
+formal definition for Web Hooks. This specification aims to provide such a
+definition for use with [CNCF CloudEvents][CE], but is considered generally
+usable beyond the scope of CloudEvents.
+
+## Status of this document
+
+This document is a working draft.
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+- 1.1. [Conformance](#11-conformance)
+- 1.2. [Relation to HTTP](#12-relation-to-http)
+2. [Delivering notifications](#2-delivering-notifications)
+3. [Authorization](#3-authorization)
+4. [Abuse Protection](#4-abuse-protection)
+5. [References](#5-references)
+
+## 1. Introduction
+
+"Web hooks" are a popular pattern to deliver notifications between applications
```

Comment:
Suggest mentioning Jeff Lindsay's post on this: http://progrium.com/blog/2007/05/03/web-hooks-to-revolutionize-the-web/. He coined it as "Web Hooks" then but since has also adopted the term "Webhook".